pyvrp/cpp/Individual.h

```diff
@@ -10,18 +9,20 @@
 
 class Individual
 {
+
+    // Outputs an individual into a given ostream in VRPLIB format
+    friend std::ostream &operator<<(std::ostream &out, Individual const &indiv);
```

Comment:
This no longer has to be a friend now that `distance()` is available. Can we revert this to its previous position below the class?

Reply:
(I'm pretty sure we can now "unfriend" `std::hash` as well, but that's out of scope for this PR)